projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.ts

```diff
@@ -15,22 +15,24 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Component, OnInit } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
 import { DetailRecord } from '@rero/ng-core/lib/record/detail/view/detail-record';
-import { Observable } from 'rxjs';
-import { MainTitleService } from '../../../service/main-title.service';
+import { Observable, Subscription } from 'rxjs';
 
 @Component({
   selector: 'admin-document-detail-view',
   templateUrl: './document-detail-view.component.html',
   styles: []
 })
-export class DocumentDetailViewComponent implements DetailRecord, OnInit {
+export class DocumentDetailViewComponent implements DetailRecord, OnInit, OnDestroy {
 
   /** Observable resolving record data */
   record$: Observable<any>;
 
+  /** Record subscription */
+  recordSubscription: Subscription;
```

Comment:
you can mark this as private (do not forget the _)

Reply:
redeclared as `private _recordObs` in all component.